Packages/io.chainsafe.web3-unity/Runtime/Scripts/UnityEnvironmentExtension.cs

```diff
@@ -16,7 +17,7 @@ public static class UnityEnvironmentExtensions
         /// <returns>Updated Service collection.</returns>
         public static IWeb3ServiceCollection UseUnityEnvironment(this IWeb3ServiceCollection services)
         {
-            services.UseApiAnalytics();
+            services.Replace(ServiceDescriptor.Singleton<IAnalyticsClient, CountlyAnalytics>());
```

Comment:
Why replace but not add as other services down below?

Reply:
Because it adds it if it's already there 😁